core/lib/multivm/src/glue/tracers/mod.rs

```diff
@@ -72,19 +72,28 @@ where
 {
     fn latest(
         &self,
-    ) -> Box<dyn crate::vm_latest::VmTracer<S, H::VmVirtualBlocksRefundsEnhancement>> {
-        Box::new(self.clone())
+    ) -> Rc<RefCell<dyn crate::vm_latest::VmTracer<S, H::VmVirtualBlocksRefundsEnhancement>>> {
+        Rc::new(RefCell::new(self.clone()))
+    }
+}
```

Comment:
Is this kind of api absolutely inevitable? I believe that you can return a `Box` and then turn it into `Rc<RefCell>` on the spot. Immediately returning that is probably not that great since it does not clearly communicate to the user of this api what's going on:

1. `Box<dyn T>>` signals that "here is some object on the heap, you have to complete ownership of it"
1. `Rc<RefCell<dyn T>>` signals "well, who knows, maybe you are the only one, maybe not, whatever, we will only know during runtime.

Reply:
Because of the performance issue, decided to get rid of Rc RefCell for now